source/includes/fact-mongodb30-upsert-id-example.rst

```diff
@@ -0,0 +1,7 @@
+For example, the following will raise an error:
+
+.. code-block:: javascript
+
+   db.collection.update( { "_id.authorID": 1 },
+      { "name": "Robert Frost" },
+      { upsert: true } )
```

Comment:
Imprecise.  Exactly why would the example shown raise an error?


> db.foo.update(       { "_id.authorID": 2 },       { "name": "Robert Frost" },       { upsert: true }    )
WriteResult({
	"nMatched" : 0,
	"nUpserted" : 0,
	"nModified" : 0,
	"writeError" : {
		"code" : 111,
		"errmsg" : "field at '_id' must be exactly specified, field at sub-path '_id.authorID'found"
	}
})
> db.foo.update(       { "_id.authorID": 1 },       { "name": "2Robert Frost" },       { upsert: true }    )
WriteResult({ "nMatched" : 1, "nUpserted" : 0, "nModified" : 1 })